src/scenicplus/cli/commands.py

```diff
@@ -366,19 +366,38 @@ def run_motif_enrichment_dem(
 
 def prepare_motif_enrichment_results(
         paths_to_motif_enrichment_results: List[str],
+        
         multiome_mudata_fname: pathlib.Path,
         out_file_direct_annotation: pathlib.Path,
         out_file_extended_annotation: pathlib.Path,
         out_file_tf_names: pathlib.Path,
+        
```

Comment:
Is there any particular reason why you added blank lines between the parameters? Otherwise I would delete them.